pages/docs/tools/asyncapi-cli.md

```diff
@@ -0,0 +1,137 @@
+---
+title: AsyncAPI CLI
+weight: 1
+---
+
+## Install CLI Using NPM and Node
```

Comment:
```suggestion
## Install CLI Using NPM and Node
```

Headers need to be succinct. You should have **installation** as the header by itself. Please follow the outline structure I provided in the issue for this :) 

<img width="400" alt="Screen Shot 2022-11-11 at 4 32 06 PM" src="https://user-images.githubusercontent.com/19964402/201447652-1560a983-55c8-4756-887e-4ab41ed77481.png">